charactersheet/charactersheet/viewmodels/actions_toolbar.js

```diff
@@ -5,7 +5,9 @@ function ActionsToolbarViewModel() {
 
     self.wellOpen = ko.observable(false);
 
-    self.init = function() {};
+    self.init = function() {
+        Notifications.actionsToolbar.toggle.add(self.toggleWellOpen);
```

Comment:
Great job on this PR! It must have been a lot of work. 
One quick question, though. Should this Notification have a .remove on the unload function?
Or does it not need one since it's in the init method?